spec/controllers/identify_controller_spec.rb

```diff
@@ -134,6 +135,7 @@
 
           it 'returns a token with a uuid and subject' do
             allow(Figaro.env).to receive(:client_cert_escaped).and_return('true')
+            allow_any_instance_of(Certificate).to receive(:has_eku?).and_return(true)
```

Comment:
so much stubbing 😭  --- is it possible to create a cert from the PEM below and then modify it and make a new PEM?